openstack_plugin/decorators.py

```diff
@@ -76,6 +78,15 @@ def wrapper_inner(**kwargs):
                                          **existing_resource_kwargs):
                     return
                 kwargs['openstack_resource'] = resource
+                if ignore_unexisted_resource:
```

Comment:
@0lvin not sure why you are adding this ? we already have `create_if_missing`  which will create external resource if it does not exist

For this if the resource does not exist then it just return, without doing anything ?

Reply:
Flag `ignore_unexisted_resource` is used for ignore `unexist` objects on `delete`/`stop`. It used only with `remove_interface_from_router` now.

Reply:
If I understand you correctly, `ignore_unexisted_resource` is only used inside the router resource for both stop/delete ?